docker/llm/finetune/lora/docker/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM mpioperator/intel as builder
+
+RUN mkdir /ppml && mkdir /ppml/data && mkdir /ppml/model
+ADD ./requirements.txt /ppml/requirements.txt
+
+# install pytorch 2.0.1
+RUN export http_proxy=http://child-prc.intel.com:913 && \
```

Comment:
one RUN to install all of these dependencies.
# install pytorch 2.0.1
RUN XXX && \
# install ipex and oneccl
    pip install XXX